crates/bevy_render/src/render_resource/pipeline_specializer.rs

```diff
@@ -138,7 +138,7 @@ impl<S: SpecializedMeshPipeline> SpecializedMeshPipelines<S> {
             entry: VacantEntry<
                 (MeshVertexBufferLayoutRef, S::Key),
                 CachedRenderPipelineId,
-                FixedHasher,
+                RandomState,
```

Comment:
I changed this to RandomState as it was a much less intrusive change: the alternative is adding complex generics to a ton of Entry callsites. Rendering people: does this choice matter?